### DISCUSSION TITLE
How to configure fluidsynth as midi driver on windows?

### DISCUSSION THREAD

**Discussion by ualaa1** [FluidSynth/fluidsynth#750] [Q&A] [answered]
I downloaded the latest binary available (2.1.6). I want to add a soundfont, and then make it appear as a midi driver (like Microsoft GS). There does not seem to be a decent guide online. When I opened the program, there seems to be some of error:
![bandicam 2021-01-21 22-31-53-471](https://user-images.githubusercontent.com/68984176/105455557-8c7f2980-5c38-11eb-908c-20e73c9cc3d8.png)

I don't know what to do next. Hopefully you guys can help me with this. I hope this isn't impossible to do.

**Comment by derselbst** [comment, marked as answer]
On startup, at least one MIDI input device must be present that fluidsynth's winmidi driver can connect to. Make sure you've plugged in a MIDI keyboard (or similar) and that you've installed all needed drivers.
